src/parser/esprima_cpp/esprima.h

```diff
@@ -33,19 +33,12 @@ class CodeBlock;
 
 namespace esprima {
 
-#define ESPRIMA_START_IDENT 1
-#define ESPRIMA_IS_IDENT 2
-#define ESPRIMA_IS_WHITESPACE 4
-#define ESPRIMA_IS_LINE_TERMINATOR 8
-
-extern char g_asciiRangeCharMap[128];
-
 // ECMA-262 11.2 White Space
 NEVER_INLINE bool isWhiteSpaceSlowCase(char16_t ch);
 ALWAYS_INLINE bool isWhiteSpace(char16_t ch)
 {
     if (LIKELY(ch < 128)) {
-        return g_asciiRangeCharMap[ch] & ESPRIMA_IS_WHITESPACE;
```

Comment:
Do you have any special reason of changing this?
I think there is no guarantee your change is faster.
You can show benchmark result in arm with large script for this.
If your change is better, we can comment it in this line for future.
I think we needs benchmark result in arm with large script for this.